### ISSUE TITLE
Log loss for regression

### ISSUE THREAD

**Issue by miguelgfierro** [Microsoft/LightGBM#263]
I'm trying to replicate in LightGBM this code in XGBoost using the python API:

```python
clf = xgb.XGBRegressor()
clf.fit(trn_x, trn_y, eval_set=[(val_x, val_y)], verbose=True, eval_metric='logloss', early_stopping_rounds=100)
```
How can I implement the evaluation metric log loss?

**Comment by guolinke** [comment]
@miguelgfierro 
"eval_metric=binary_logloss"
